### PR TITLE
chore(swap): add createdAt to status

### PR DESCRIPTION
**Pull request by zoheb391**

### CODE REVIEW

File: packages/swap/src/routes/swap.ts [hunk 1]
```diff
@@ -129,6 +129,7 @@ router.get(
       srcAsset,
       destAsset,
       destAddress: readField('destAddress'),
+      createdAt: swapDepositChannel?.createdAt.valueOf(),
```
Comment (niklasnatter) [inline packages/swap/src/routes/swap.ts:132]: might be a bit confusing as its not defined for smart contract calls
```suggestion
      depositAddressCreatedAt: swapDepositChannel?.createdAt.valueOf(),
```

**Review by niklasnatter — COMMENTED** [review]
need to add this to the sdk types to be able to use it

File: packages/swap/src/routes/swap.ts (hunk 1 above)
Comment (zoheb391) [reply packages/swap/src/routes/swap.ts:132]: good point 🤦

File: packages/sdk/src/swap/types.ts [hunk 2]
```diff
@@ -62,13 +62,15 @@ export type SwapStatusResponse = CommonStatusFields &
         swapId: string;
         depositAmount: string;
         depositReceivedAt: number;
+        depositAddressCreatedAt: number;
```
Comment (niklasnatter) [inline packages/sdk/src/swap/types.ts:65]: think you can move it to `CommonStatusFields` 🙂

Comment (niklasnatter) [reply packages/sdk/src/swap/types.ts:65]: and it needs to be `number | undefined` i guess

Comment (niklasnatter) [reply packages/sdk/src/swap/types.ts:65]: its not great that we dont check if the response matches the types we have anywhere 🙈

**Review by acdibble — APPROVED** [review]

**Review by niklasnatter — APPROVED** [review]

**Pull request merged**